Content.Server/StationEvents/Metric/DoorMetric.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Content.Server.Chemistry.Components.SolutionManager;
+using Content.Server.Chemistry.EntitySystems;
+using Content.Server.Power.Components;
+using Content.Server.StationEvents.Metric.Components;
+using Content.Shared.Doors.Components;
+using Content.Shared.FixedPoint;
+using Content.Shared.Fluids.Components;
+
+namespace Content.Server.StationEvents.Metric;
+
+/// <summary>
+///   Uses doors and firelocks to sample station chaos across the station
+///
+///   Emag - 10 points per emaged door
+///   Power - 5 points per door or firelock with no power
+///   Atmos - 10 points for holding spacing or 20 for holding back fire
+/// </summary>
```

Comment:
Don't duplicate the values here because they're going to be wrong if any changes are made.